lib/ecs_deploy/auto_scaler/cluster_resource_manager.rb

```diff
@@ -77,7 +77,9 @@ def trigger_capacity_update(old_desired_capacity, new_desired_capacity, interval
         th = Thread.new do
           @logger&.info "#{log_prefix} Start updating capacity: #{old_desired_capacity} -> #{new_desired_capacity}"
           Timeout.timeout(180) do
-            until @capacity == new_desired_capacity || (new_desired_capacity >= old_desired_capacity && @capacity > new_desired_capacity)
+            until @capacity == new_desired_capacity ||
+                (new_desired_capacity >= old_desired_capacity && @capacity > new_desired_capacity) ||
```

Comment:
[Q] Why `new_desired_capacity >= old_desired_capacity` (greater or **equal**) in 2nd condition while `new_desired_capacity < old_desired_capacity` (less than) in 3rd condition?

Is there any problem if we change this with like `(new_desired_capacity > old_desired_capacity && @capacity > new_desired_capacity)`?

Reply:
Currently, this method is never called with the same value of `new_desired_capacity` as that of `old_desired_capacity`. However, in the following case, the condition is never met:

* `old_desired_capaacity` is 101
* `new_desired_capaacity` is 101
* All instances have 2 vCPUs
* `@capacity`: 102

I've refactored the code to make it more readable: https://github.com/reproio/ecs_deploy/pull/80/commits/7862fefee9fefab058d41d7cd106ffcf011a9b6c